test/framework/easyconfig.py

```diff
@@ -2094,6 +2094,36 @@ def test_dump(self):
                 if param in ec:
                     self.assertEqual(ec[param], dumped_ec[param])
 
+        ec_txt = textwrap.dedent("""
+            easyblock = 'EB_toy'
+
+            name = 'foo'
+            version = '0.0.1'
+
+            toolchain = {'name': 'GCC', 'version': '4.6.3'}
+
+            homepage = 'http://foo.com/'
+            description = "foo description"
+
+            sources = [SOURCE_TAR_GZ]
+            source_urls = ["http://example.com"]
+
+            dependencies = [
+                ('toy', '0.0', '', True),
```

Comment:
Would we squeeze in another dep that also uses the SYSTEM variable here as well to be sure nothing surprising happens

Reply:
Yeah, good idea, I'll look into that

Reply:
@Micket fixed in f8f1019fa